apps/README.md

```diff
@@ -37,7 +37,7 @@ npm run build
 ```
 and look for something like:
 ```
-lrwxr-xr-x  1 laurel  501  12 Apr 27 13:00 dashboard/public/blockly -> apps/build/package
+lrwxr-xr-x  1 laurel  501  12 Apr 27 13:00 dashboard/public/blockly -> apps-package
```

Comment:
If it's pointing at apps-package instead of apps/build/package, it means you're using the prebuilt version instead of your personally built copy of apps. The way the instructions are written, I think they expect it to point at apps/build/package as written.